### DISCUSSION TITLE
Installing an error in the nextjs project, I am not sure whether the current version can be used in the production environment

### DISCUSSION THREAD

**Discussion by chaocwu** [chakra-ui/ark#2563] [General]
### Description

Installing an error in the nextjs project, I am not sure whether the current version can be used in the production environment

### Link to Reproduction (or Detailed Explanation)

not involving

### Steps to Reproduce

pnpm install @ark-ui/react

### Ark UI Version

latest

### Framework

- [X] React
- [ ] Solid
- [ ] Vue

### Browser

Edge

### Additional Information

![image](https://github.com/chakra-ui/ark/assets/143390961/78b9e4f7-8ef4-4d25-8aa8-a8d42ad5228f)

**Comment by segunadebayo** [comment]
Can you try again with the latest release?

**Comment by segunadebayo** [comment]
Closing — this was likely version-specific and has since been resolved. Feel free to reopen if you are still experiencing issues.
